docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3.4'
+
+volumes:
+  bds:
+    external: true
+
+services:
+
+  bds1:
+    container_name: emer
+
+    image: itzg/minecraft-bedrock-server
+    environment:
+      SERVER_NAME: "EmerLand"
+      LEVEL_NAME: "EmerLand"
+      # "nice village near spawn"
+      # LEVEL_SEED: "720057903"
+      MAX_PLAYERS: 20
+      EULA: "TRUE"
+      GAMEMODE: adventure
+      DIFFICULTY: normal
+      ALLOW_CHEATS: "true"
+      WHITE_LIST: "true"
+    ports:
+      - 19132:19132/udp
+      - 39132:39132
+      - 39132:39132/udp
+      - 39133:39133
+      - 39133:39133/udp
+    sysctls:
+      net.ipv4.ip_local_port_range: 39132 39133
```

Comment:
FYI, when I added this to my local copy the server zip download failed with:
```
bds_1  | DEBU[0000] Using /data to match uid and gid             
bds_1  | DEBU[0000] Resolved UID=0 from match path               
bds_1  | DEBU[0000] Resolved GID=0 from match path               
bds_1  | Failed to extract download URL '' from https://www.minecraft.net/en-us/download/server/bedrock/
bds_1  | 2020/07/15 23:17:12 Failed to get from URLGet https://www.minecraft.net/en-us/download/server/bedrock/: dial tcp: lookup www.minecraft.net on 127.0.0.11:53: read udp 127.0.0.1:39133->127.0.0.11:53: i/o timeout
examples_bds_1 exited with code 2
```

So, I think I'll leave the example compose file as is.